be/src/formats/orc/orc_file_writer.cpp

```diff
@@ -599,6 +551,20 @@ StatusOr<std::unique_ptr<orc::Type>> ORCFileWriter::_make_schema_node(const Type
     }
 }
 
+void ORCFileWriter::_populate_orc_notnull(orc::ColumnVectorBatch& orc_column, uint8_t* null_column,
```

Comment:
can it be a static function? no need to be a method of class

Reply:
done